src/atoms/button/Button.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from "prop-types";
+
+import { ButtonStyled } from "./Button.styled";
+
+const Button = ({ disabled, text, click }) => {
+  return (
+    <ButtonStyled onClick={click} disabled={disabled}>
+      {text}
+    </ButtonStyled>
+  );
+};
+
+Button.propTypes = {
+  click: PropTypes.func,
+  disabled: PropTypes.bool,
+  text: PropTypes.string,
+};
+
+Button.defaultProps = {
+  click: () => alert("Button"),
```

Comment:
by default it should be () => {}
As if no onClick will be passed user would see Button 

Also, rename click to either handleClick or onClick as simply click is not indicate event handler